src/components/Currencies.js

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { FaChevronDown, FaChevronUp } from 'react-icons/fa';
+import './Currencies.css';
+
+const Currencies = () => {
```

Comment:
- [ ] **[OPTIONAL]** I suggest you improve the color combination of the app, as your current one stresses vision.